build/conf/nsis/idea.nsi

```diff
@@ -617,7 +617,7 @@ Function silentConfigReader
   ${LogText} "  config file: $R1"
 
   ${ConfigRead} "$R1" "mode=" $R0
-  IfErrors no_silent_config
+  IfErrors bad_silent_config
   ${LogText} "  mode: $R0"
   StrCpy $silentMode "user"
   IfErrors launcher_32
```

Comment:
(not related to the change, just a question) The above two lines are pointless, aren't they? The NSIS script syntax looks as if it was created by aliens for predators, so I'll really appreciate a second opinion.

Reply:
They seem useless, but they might be checking if `$silentMode` is not defined. I would need to test that.

Reply:
I see, thanks. I'll leave it as is for now.